webapp/app/js/filters.js

```diff
@@ -39,11 +39,21 @@ treeherder.filter('stripHtml', function() {
 
 treeherder.filter('linkifyBugs', function() {
     return function(input) {
-        var re = new RegExp('(?:Bug (\\d+))', 'ig');
-        var str = input || '';
-        return str.replace(re,
-            '<a href="https://bugzilla.mozilla.org/show_bug.cgi?id=$1" target="_blank" title="Bug $1">$1</a>'
-        );
+        if (input) {
+            var re = /.*-- (bug-)?([0-9]+)/i; //regex for matching bug number
+            var bug_matches = re.exec(input);
```

Comment:
Also, I don't think this will work for the old case where we have "Bug 12345 - Fix the foo" which is the style used in mozilla-central. If you want help crafting a regular expression that will work for that case feel free to ask in #ateam or #treeherder. :)